src/main/java/domain/LottoResult.java

```diff
@@ -0,0 +1,59 @@
+package domain;
+
+import java.text.DecimalFormat;
+import java.util.Arrays;
+import java.util.List;
+
+public enum LottoResult {
+    NO_WIN(0, -1, false),
+    FIRST(2000000000, 6, false),
```

Comment:
```suggestion
    FIRST(2_000_000_000, 6, false),
```

언더스코프를 활용하여 가독성을 상승시킬 수 있습니다!

Reply:
훨씬 보기가 편하네요! 감사합니다 👍